pkg/storage/mvcc.go

```diff
@@ -5332,6 +5332,144 @@ func CanGCEntireRange(
 	return coveredByRangeTombstones, nil
 }
 
+var emptyMetaToken enginepb.MVCCMetadata
+
+// MVCCGarbageCollectPointsWithClearRange removes garbage collected data falling
+// under ranges covered by [start@startTimestamp, endTimestamp). This function
```

Comment:
This comment doesn't appear accurate, it doesn't matter whether it's below a range key or not?